carbon_tracker_workflow.ipynb

```diff
@@ -0,0 +1,2510 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ds_input = carbon_tracker.merge_datasets(desired_data, ds_cb)
</pre><p>This now does loading and merging... Perhaps it's not needed to wrap that in a function, as it's already quite simple to load and marge.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EXCITED-CO2/excited-workflow/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>